pkg/kubelet/kubelet.go

```diff
@@ -1386,7 +1386,11 @@ func (kl *Kubelet) GetKubeletContainerLogs(podFullName, containerName, tail stri
 		// waiting state.
 		return err
 	}
-	return kl.containerManager.GetContainerLogs(containerID, tail, follow, stdout, stderr)
+	pod, ok := kl.GetPodByFullName(podFullName)
```

Comment:
We do today to get the containerID, I think we may be able to remove that need with using the container name. I'll add a TODO, it's not super clear to me what to do as of yet though.